scripts/ci/test-libs.sh

```diff
@@ -1,4 +1,39 @@
 #!/usr/bin/env bash
 export NODE_ENV="ci"
 
-ng test --browsers ChromeHeadless --watch=false --code-coverage
+# List of all projects to test
+PROJECTS=(
+    "ppwcode"
+    "@ppwcode/ng-async"
```

Comment:
Does this have to be in a specific order? Preferably this is sorted alphabetically.